.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+# instruct GitHub dependabot to scan github actions for updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    # dependabot automatically checks .github/workflows/ and .github/actions/
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "pip"
+    # we (for better or worse) have dumped the python code in the root of the repo
```

Comment:
🤨

Reply:
I'm still handling python about as well as a toddler handles a sippy cup. For the most part, everything works as intended, but sometimes there's a lot of fumbling around and crying.